engine/classes/Elgg/Database/LegacyQueryOptionsAdapter.php

```diff
@@ -1020,18 +994,10 @@ protected function normalizeGroupByClauses(array $options = []) {
 		}
 
 		if (is_string($options['group_by'])) {
-			$clause = $options['group_by'];
-
 			$options['group_by'] = explode(',', $options['group_by']);
 
 			if (count($options['group_by']) > 1) {
```

Comment:
@hypeJunction why are we only concerned if more than 1?

Reply:
We can deal with one group by query, multiple queries assume something can be recorded

Reply:
Reordered*